addins/Cake.Wyam.Recipe.yml

```diff
@@ -0,0 +1,12 @@
+Name: Cake.Wyam.Recipe
```

Comment:
Should be filtered as it is not an addin

Reply:
Done. Packages containing "recipes" will be filtered out from YAML generation